src/infrastructure/gateway/generalPartner/GeneralPartnerGateway.ts

```diff
@@ -4,7 +4,7 @@ import { SDK_LIMITED_PARTNERSHIP_SERVICE } from "../../../config/constants";
 import { Resource } from "@companieshouse/api-sdk-node";
 import {
   GeneralPartner,
-  LimitedPartnershipResourceCreated
+  LimitedPartnershipResourceCreated,
```

Comment:
I don't think all these additional commas need adding. There's a lot of them.

See [this](https://companieshouse.slack.com/archives/G01JK7RBSTB/p1741702915862609?thread_ts=1741701000.847379&cid=G01JK7RBSTB) from @lduranteau.